src/promptflow-tools/promptflow/tools/list.py

```diff
@@ -1,13 +1,14 @@
 from pathlib import Path
 
-import yaml
+from ruamel.yaml import YAML
 
 
 def collect_tools_from_directory(base_dir) -> dict:
     tools = {}
+    yaml = YAML()
     for f in Path(base_dir).glob("**/*.yaml"):
         with open(f, "r") as f:
-            tools_in_file = yaml.safe_load(f)
+            tools_in_file = yaml.load(f)
```

Comment:
Do we need to include the 'ruamel' in requirements.txt of the tool package?

Reply:
`ruamel.yaml `is a dependency of `promptflow`, and `promptflow` is a dependency of `promptflow-tools`, so I think the tool already indirectly depends on `ruamel. yaml` and we donot need to add it to requirements.txt.